.vscode/extensions.json

```diff
@@ -1,7 +1,7 @@
 {
   "recommendations": [
     "dbaeumer.vscode-eslint",
-    "esbenp.prettier-vscode",
```

Comment:
This was deprecated in favor of the new extension (unrelated to the rest of this PR)